prime-router/docs/getting-started/running-tests.md

```diff
@@ -25,6 +25,18 @@ cd ./prime-router
 # Or to force the unit tests to run
 ./gradlew test -Pforcetest
 ```
+
+## Running smaller sets of unit tests
```

Comment:
Could we also add a note on running test groups a la:

```
./prime test --run=server2serverauth
```

(or with its gradlew equivalent primeCLI).

Reply:
This comment inspired me to look over this doc file with fresh eyes and I ended up making a host of changes. If you have a moment to give this doc another once over as a sanity check that would be great: https://github.com/CDCgov/prime-reportstream/blob/platform/josh/14332-FHIRRouter-store-messageID/prime-router/docs/getting-started/running-tests.md

Reply:
Looks awesome 👍